### PR TITLE
In-flight not outstanding

### PR DESCRIPTION
**Pull request by ianswett**
Issue noticed when reviewing PR #2117

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -400,7 +400,7 @@ On each consecutive expiration of the crypto timer without receiving an
 acknowledgement for a new packet, the sender SHOULD double the crypto
 retransmission timeout and set a timer for this period.
 
-When crypto packets are outstanding, the TLP and RTO timers are not active.
+When crypto packets are in-flight, the TLP and RTO timers are not active.
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
When crypto packets are in flight, the TLP and RTO timers are not active.
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -436,9 +436,9 @@ conditions:
   PTO SHOULD be scheduled for min(RTO, PTO).
 
 QUIC includes MaxAckDelay in all probe timeouts, because it assumes the ack
-delay may come into play, regardless of the number of packets outstanding.
-TCP's TLP assumes if at least 2 packets are outstanding, acks will not be
-delayed.
+delay may come into play, regardless of the number of packets ack-eliciting
+packets in-flight. TCP's TLP assumes if at least 2 ack-eliciting packets are
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
packets in flight. TCP's TLP assumes if at least 2 ack-eliciting packets are
```

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -436,9 +436,9 @@ conditions:
   PTO SHOULD be scheduled for min(RTO, PTO).
 
 QUIC includes MaxAckDelay in all probe timeouts, because it assumes the ack
-delay may come into play, regardless of the number of packets outstanding.
-TCP's TLP assumes if at least 2 packets are outstanding, acks will not be
-delayed.
+delay may come into play, regardless of the number of packets ack-eliciting
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
delay may come into play, regardless of the number of ack-eliciting
```

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -436,9 +436,9 @@ conditions:
...
+packets in-flight. TCP's TLP assumes if at least 2 ack-eliciting packets are
+in-flight, acks will not be delayed.
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
in flight, acks will not be delayed.
```

File: draft-ietf-quic-recovery.md [hunk 5]
```diff
@@ -796,7 +796,7 @@ Pseudocode for SetLossDetectionTimer follows:
       loss_detection_timer.cancel()
       return
 
-    if (crypto packets are outstanding):
+    if (crypto packets are in-flight):
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
    if (crypto packets are in flight):
```

File: draft-ietf-quic-recovery.md [hunk 6]
```diff
@@ -837,7 +837,7 @@ Pseudocode for OnLossDetectionTimeout follows:
 
 ~~~
    OnLossDetectionTimeout():
-     if (crypto packets are outstanding):
+     if (crypto packets are in-fight):
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
     if (crypto packets are in flight):
```

File: draft-ietf-quic-recovery.md [hunk 7]
```diff
@@ -1043,7 +1043,7 @@ kMaxDatagramSize:
   windows. The RECOMMENDED value is 1200 bytes.
 
 kInitialWindow:
-: Default limit on the initial amount of outstanding data in bytes.  Taken from
+: Default limit on the initial amount of in-flight data in bytes.  Taken from
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
: Default limit on the initial amount of data in flight, in bytes.  Taken from
```

**Comment by ianswett** [comment]
@janaiyengar Suggestions accepted.

**Pull request merged**